test-packages/integration-tests/test/cache.spec.ts

```diff
@@ -110,72 +110,33 @@ describe('CacheDestination & CacheClientCredentialToken', () => {
     delete process.env['VCAP_SERVICES'];
   });
 
-  it('caches the Destinations service call which fixed isolation Tenant (only simple auth flows)', async () => {
-    await populateDestinationsServiceCache();
+  it('caches the destinations service call with fixed isolation tenant (only simple auth flows)', async () => {
+    const retrieveDestination = isolationStrategy =>
+      getDestination({
+        destinationName: 'FINAL-DESTINATION',
+        useCache: true,
+        isolationStrategy
+      });
+
+    await retrieveDestination('tenant');
```

Comment:
[req] The func call is not needed here.

Reply:
Technically yes. Honestely, I don't really get the test setup... Let's discuss this.